chain/chain/src/state_snapshot_actor.rs

```diff
@@ -107,6 +107,10 @@ impl StateSnapshotActor {
         msg: CreateSnapshotRequest,
         ctx: &mut dyn DelayedActionRunner<Self>,
     ) {
+        if let StateSnapshotConfig::Disabled = self.tries.state_snapshot_config() {
```

Comment:
ditto about comparison instead of pattern matching

Reply:
Ditto as well.